readme.md

```diff
@@ -14,42 +14,46 @@ $ npm install electron-main-fetch
 
 ## Usage
 
-The difference between this and `Fetch` is that instead of receiving a [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response) object and calling a method on it for what type to receive, you just specify the type in the function call.
-
 ```js
 const fetch = require('electron-main-fetch');
 
-(async () => {
-	const ip = await fetch('https://api.ipify.org', {type: 'text'});
-	console.log(ip);
-	//=> '170.56.15.35'
-})();
-```
-
-With `Fetch` in the renderer process, the above would instead be:
-
-```js
 (async () => {
 	const response = await fetch('https://api.ipify.org');
-	const ip = await response.text();
-	console.log(ip);
+	console.log(await response.text());
 	//=> '170.56.15.35'
 })();
 ```
 
+The main difference between this and `Fetch` is that we don't have direct access to the body stream, so there is no `body` property.
+There is some methods that are not useful in Node.js like `formData()`, `redirect()`, etc.
```

Comment:
This should explicitly document which methods that are missing. And any other difference, if any.